docs/install-and-use.md

```diff
@@ -53,6 +53,24 @@ To make use of the components, you'll have to include the JS and CSS in your app
 
 ### CSS
 
+The Sass for any components in use in an application should be imported. In order for this to work you must first import some generic Sass files that include `govuk-frontend` and some gem specific mixins and variables.
+
+Note that you will need to keep this up to date if you add or remove components from your application.
+
+For example:
+
+```
```

Comment:
```suggestion
```scss
```

Reply:
Thanks! Done. Also you win the prize for 'fastest unsolicited review ever' 😁